### PR TITLE
updated available algorithms list

### LINKED ISSUE #83: List Available Algorithms of the Module
As the Module is constantly upgrading, a list of implemented algorithms needs to be present at the cover ReadMe of the repository. It will help Users to find and select algorithms they need and will help contributors to decide to write a new algorithm for the module.

### PR DESCRIPTION
**Pull request by ravi5175**
### Description
Added a list of available algorithms in module to the readme

Fixes #83

### Checklist:
- [* ] I read the **CONTRIBUTING** document before making this pull request
- [* ] My code follows the style guidelines of this project
- [*] I have performed a self-review of my own code or materials
- [* ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [*] My change requires change to documentation
- [* ] I have updated documentation accordingly
- [* ] My changes generate no new warnings 
- [* ] All new and existing tests passed

### CODE REVIEW

**Review by codesankalp — CHANGES_REQUESTED** [review]
With links to the file will be better.

**Comment by ravi5175** [comment]
@codesankalp commited requested changes

File: README.rst [hunk 1]
```diff
@@ -35,11 +35,36 @@ To cover different available data structures and algorithms for generic cases.
 
 ----------------------------------
 
-**CURRENT FEATURES**
-====================
-
-* linked list
-* sorting algorithms
+** ALGORITHMS AVAILABLE**
```
Comment (codesankalp) [inline README.rst]:
```suggestion
**ALGORITHMS AVAILABLE**
```

File: README.rst [hunk 2]
```diff
@@ -35,11 +35,36 @@ To cover different available data structures and algorithms for generic cases.
...
+==========================
+
+* Linked List algorithms `link <https://github.com/codesankalp/dsalgo/blob/master/dsalgo/linked_list.py>`_
```
Comment (codesankalp) [inline README.rst]:
```suggestion
* `Linked List algorithms <https://github.com/codesankalp/dsalgo/blob/master/dsalgo/linked_list.py>`_
```
And for other also.

**Review by codesankalp — CHANGES_REQUESTED** [review]
@ravi5175  I have reviewed it, It will be better if the headline will be link.

File: README.rst (hunk 2 above)
Comment (ravi5175) [reply README.rst]: > @ravi5175 I have reviewed it, It will be better if the headline will be link.

sure i will fix

**Review by codesankalp — APPROVED** [review]

**Review by Aryamanz29 — APPROVED** [review]

**Pull request merged**